modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,182 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+----
++
+This feature is part of the TEP-0076 and is in progress to index into the array result while consuming that result.
+// link:https://github.com/tektoncd/pipeline/pull/4818
+
+[id="triggers-new-features-1-8_{context}"]
+=== Triggers
+
+* This update transitions the `TriggerGroups` field in the `EventListener` specification from being an alpha feature to being a stable feature. Using this field, you can specify a set of interceptors before selecting and running a group of triggers. It is available by default, and you do not need to change the configuration to use it.
+// link:https://github.com/tektoncd/triggers/pull/1345(#1345)
+// @dlorenc
+
+* With this update, the `Trigger` resource supports end-to-end secure connections by running the `ClusterInterceptor` server using HTTPS.
+// link:https://github.com/tektoncd/pipeline/pull/1333(#1333)
+// @savitaashture
+
+[id="cli-new-features-1-8_{context}"]
+=== CLI
+
+* With this update, you can use the `tkn taskrun export` command to export a live task run from a cluster to a YAML file, which you can use to import the task run to another cluster.
+// link:https://github.com/tektoncd/cli/pull/1531[(#1531)]
+// Chmouel Boudjnah @chmouel
+
+* With this update, you can add the `-o name` flag to the `tkn pipeline start` command to print the name of the pipeline run right after it starts.
+// link:https://github.com/tektoncd/cli/pull/1540[(#1540)]
+// Chmouel Boudjnah @chmouel
+
+* This updates adds the list of available plugins to the `tkn --help` command.
+// link:https://github.com/tektoncd/cli/pull/1535[(#1535)]
+// Chmouel Boudjnah @chmouel
+
+* With this update, while deleting a pipeline run or task run, you can use both the `--keep` and `--keep-since` flags together.
+// link:https://github.com/tektoncd/cli/pull/1533[(#1533)]
+// Divyanshu Agrawal @divyansh42
+
+* With this update, you can use the `spec.status` value, `Cancelled`, instead of the deprecated `PipelineRunCancelled` status.
+// link:https://github.com/tektoncd/cli/pull/1554[(#1554)]
+// Vinamra Jain @vinamra28
+
+[id="operator-new-features-1-8_{context}"]
+=== Operator
+
+// [Hub]
+* With this update, as an administrator, you can configure your local Tekton Hub instance to use a custom database rather than the default database.
+// (link:https://issues.redhat.com/browse/SRVKP-2309[SRVKP-2309 Pipeline as code: validate repository URL uniqueness])
+// Puneet Punamiya
+
+* With this update, as a cluster administrator, if you enable your local Tekton Hub instance, it periodically refreshes the database so that changes in the catalog appear in the Tekton Hub web console. You can adjust the period between refreshes.
++
+Previously, to add the tasks and pipelines in the catalog to the database, you performed that task manually or set up a cron job to do it for you.
+// (link:https://issues.redhat.com/browse/SRVKP-2252[SRVKP-2252 Refresh catalogs on schedule])
+// Puneet Punamiya
```

Comment:
@PuneetPunamiya Please review and approve. Thanks.

Reply:
This feature is new and looks good to me 👍🏻